src/compositemark/boxplot.ts

```diff
@@ -286,6 +286,10 @@ export function normalizeBoxPlot(
       filteredLayersMixins = filteredWhiskerSpec;
       filteredLayersMixins.transform.unshift(joinaggregateTransform);
     }
+    filteredLayersMixins.transform = [
```

Comment:
`filteredLayersMixins` can be undefined here --  better streamline the logic a bit more.   Plus, you can just call `unshift` like above.

Reply:
In fact, better just include this as a part of all joinAggregateTransform above -- something like: 

`unshift(preJoinAggregate, joinAggregate)`

Reply:
I think `unshift(preJoinAggregate, joinAggregate)` might not work in this case because `preJoinAggregate` should be an array since `joinAggregate` might depend on more than one transform.

Reply:
`unshift(...preJoinAgg, joinAgg)`

Reply:
Ok, thank you.